_posts/2016-01-21-measuring-episerver-site-performance.md

```diff
@@ -0,0 +1,83 @@
+You can download the GitHub repository as a [ZIP file](https://github.com/dratini/jmeter-perfotrator/archive/master.zip) for convenience, and it should work right out-of-the box, though Java is required, sorry for that.
+
+First, grab some urls for testing. Add them to file [_project\urls.txt_](https://github.com/dratini/jmeter-perfotrator/blob/master/project/urls.txt), one per line. 
+Then fire up JMeter with _start-gui.bat_, you should now see the JMeter UI.
+It might look like a Soviet nuclear plant control panel, but the only buttons you need to know for basic use are these.
+
+[![JMeter basic controls](/img/measuring-episerver-site-performance/jmeter-basic-controls.jpg)](/img/measuring-episerver-site-performance/jmeter-basic-controls.jpg)
+
+Eager as we are, let us fire up the test by pressing START, and view the results as Aggregate Report. In this test I used Episerver Alloy demo on localhost.
+
+[![JMeter report view](/img/measuring-episerver-site-performance/jmeter-report-view.jpg)](/img/measuring-episerver-site-performance/jmeter-report-view.jpg)
+
+The report is easy to interpret, just ensure that Error rate is always zero. 
+The most interesting column is Throughput as it's unit is requests per second, the answer we were looking for.
+Please note that only the html is downloaded; stylesheets, scripts and other embedded resources are ignored.
+
+While executing the test, the server CPU usage should be sky-high.
+
+![Maximize CPU usage](/img/measuring-episerver-site-performance/metalcrusher.jpg)
+
+If not, you can change the execution parameters to increase the load.
+By default, the project uses 15 threads for requests, that is 15 simultaneous requests running at a time. 
+Each thread will execute 100 requests, totalling to 1500 requests per each url.
+The defaults are changeable on _Test Plan_-node.
+
+![Change execution parameters](/img/measuring-episerver-site-performance/jmeter-your-best-nightmare.jpg)
+
+While we are at it, let us see the other options too.
+
+![Change execution parameters](/img/measuring-episerver-site-performance/jmeter-other-confs.jpg)
+
+You can enable grayed out nodes by left-clicking and choosing _Enable_ in the context menu.
+
+* **Cookie Manager**: Enables cookies. Threads will remember and use cookies like a browser. You can also add custom cookies that are used by all threads, for example, a session cookie for authentication.
+* **View Results in Table**: Shows details per request in a table. Not as verbose as View Results in Tree. 
+* **View Results in Tree**:Very detailed results per request. Contains http request and response details, thus making this very useful while debugging site access and errors.
+
+### Reliable site setup
+The most reliable results are acquired from an environment identical to production, unfortunately this is not always possible in development stage. 
```

Comment:
I'd add full stop after "production", so "The most reliable results are acquired from an environment identical to production. Unfortunately this is not always possible in development stage." makes it more natural to read. Or maybe add "but" there.